includes/Core/Assets/Script.php

```diff
@@ -69,7 +69,23 @@ public function register( Context $context ) {
 		$version = $this->args['version'];
 
 		if ( $src ) {
-			list( $filename, $hash ) = Manifest::get( $this->handle );
+			$entry = Manifest::get( $this->handle );
```

Comment:
I did look into writing some PHPUnit tests for these changes, but discovered there is no existing coverage for the execution path where the entry is retrieved from `Manifest`, presumably because `Manifest` is designed to load from `dist/manifest.php` which makes it tricky to use in tests. We could change things so `Manifest` is an injected dependency but I did not feel comfortable with the amount of changes that would be necessary to make this testable.